code/modules/food_and_drinks/recipes/recipes_microwave.dm

```diff
@@ -52,24 +55,49 @@
 /datum/recipe/microwave/human/burger
 	items = list(
 		/obj/item/food/snacks/meat/human,
+		/obj/item/food/snacks/grown/lettuce,
 		/obj/item/food/snacks/bun
 	)
 	result = /obj/item/food/snacks/human/burger
 
 /datum/recipe/microwave/plainburger
 	items = list(
 		/obj/item/food/snacks/bun,
+		/obj/item/food/snacks/grown/lettuce,
 		/obj/item/food/snacks/meat //do not place this recipe before /datum/recipe/microwave/humanburger
```

Comment:
While we're here, could you quickly update this comment to say the correct typepath?

Reply:
Done.